concepts/index.md

```diff
@@ -185,6 +185,27 @@ day.
 * when using the service ___one_day___ or ___one_day_early___ the parameter
 ___package.description___ is mandatory
 
+### DHL Express
+
+#### Shipments with Pickup Requests
+If you don't have regular pickups by DHL Express you will have to request pickups by the carrier.
```

Comment:
I feel like this and the following sentence could be a bit ambiguous. I suggest something like:

`"If you don't have regular pickups by DHL Express you will have to request a pickup when creating a shipment. Therefore, you can add a pickup element to your shipment request."`

What do you think?